Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -1038,7 +1038,7 @@ def test():
     elif demisto.command() == 'rs-related-incidents':
         demisto.results(related_incidents_command(demisto.args()['incident-id']))
 
-except Exception, e:
+except Exception as e:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.